extension/src/plots/vega/util.test.ts

```diff
@@ -264,3 +265,44 @@ describe('extendVegaSpec', () => {
     expect(updatedSpecString).toContain(truncatedTitle)
   })
 })
+
+describe('reverseOfLegendSuppressionUpdate', () => {
+  it('should reverse the legend suppression applied by extendVegaSpec', () => {
+    type NonOptionalEncoding = { [P in keyof Encoding]-?: Encoding[P] }
```

Comment:
[F] This means that if the Encoding property is updated then compilation will fail unless this test is also updated.

Reply:
Which should mean we don't miss the required webview update.